README.md

```diff
@@ -31,68 +32,76 @@
        transcribe  Transcribe one or more audio file(s)
    ```
 
+
 ## Example usage
 
 - A normal real time session using a .wav file as the input audio
 
    ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ speechmatics transcribe--url $URL --lang en --ssl-mode=none example_audio.wav
+   $ URL=ws://realtimeappliance.yourcompany:9000/v2
+   $ speechmatics transcribe --url $URL --lang en --ssl-mode none example_audio.wav
    ```
 
-- A normal real time session with a locally running container
+- Show the messages that are going over the websocket connection
 
    ```shell
-   $ URL=ws://127.0.0.1:9000/v2
-   $ speechmatics transcribe --url $URL --lang en --ssl-mode=none example_audio.wav
+   $ speechmatics transcribe -v --url $URL --lang en --ssl-mode none example_audio.wav
    ```
 
-- Show the messages that are going over the websocket connection
+- The CLI also accepts an audio stream on standard input; transcribe the piped input audio
 
    ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ speechmatics transcribe -v --url $URL --lang en --ssl-mode=none example_audio.wav
+   $ cat example_audio.wav | speechmatics transcribe --ssl-mode none --url $URL --lang en -
    ```
 
-- Similar to the first example, but this time the input audio is piped in
+- Pipe audio directly from the microphone (example uses MacOS with [ffmpeg](https://ffmpeg.org/ffmpeg-devices.html#avfoundation))
 
-   ```shell
-   $ URL=ws://realtimeappliance.mycompany.io:9000/v2
-   $ cat example_audio.wav | speechmatics transcribe --ssl-mode=none --url $URL --lang en -
-   ```
+    First list available input devices with
 
-- The CLI also accepts an audio stream on standard input, meaning that you can stream in a live microphone feed for example.
+    ```shell
+    $ ffmpeg -f avfoundation -list_devices true -i ""
+    ```
 
-   **MacOS example with ffmpeg**
+    There needs to be at least one available microphone attached to your computer.
+    The command below gets the microphone input and pipes it to the transcriber.
+    You may need to change the sample rate to match the sample rate that your machine records at.
+    You may also need to replace `:default` with something like `:0` or `:1` if you want to use a specific microphone.
 
-   The command to list input devices available to ffmpeg is:
+    ```shell
+    $ ffmpeg -f avfoundation -i ":default" -f f32le -acodec pcm_f32le -ar 44100 - \
+    >   | speechmatics transcribe --ssl-mode none --url $URL --raw pcm_f32le --sample-rate 44100 --lang en -
+    ```
 
-   ```shell
-   $ ffmpeg -f avfoundation -list_devices true -i ""
-   ```
+- Transcribe in real-time with partials (example uses Ubuntu with ALSA)
```

Comment:
Forgive my ignorance, is "partials" something obvious to everyone or something we should explain (point to )?

Reply:
Fixed